runtime/service/npud/core/DBus.cc

```diff
@@ -132,5 +134,28 @@ gboolean DBus::on_handle_network_destroy(NpudCore *object, GDBusMethodInvocation
   return TRUE;
 }
 
+gboolean DBus::on_handle_request_create(NpudCore *object, GDBusMethodInvocation *invocation,
+                                        guint64 arg_ctx, guint arg_nw_handle)
+{
+  VERBOSE(DBus) << "on_handle_request_create with " << arg_ctx << ", " << arg_nw_handle
+                << std::endl;
+  RequestID requestID = 0;
+  int ret = -1;
+  // TODO Invoke core function.
+  npud_core_complete_request_create(object, invocation, guint(requestID), ret);
+  return TRUE;
```

Comment:
I have a curious question. Why is `ret` not returend?

Reply:
> The `return TRUE` is from `gdbus` guideline. I wanted to find the document about this, but I couldn't. 
> Dbus regards that the `ret` value of `npud_core_complete_context_create` is the return value of function. If it has an error, we need to use [g_dbus_method_invocation_return_error](https://www.freedesktop.org/software/gstreamer-sdk/data/docs/latest/gio/GDBusMethodInvocation.html#g-dbus-method-invocation-return-error) function.

from https://github.com/Samsung/ONE/pull/10026#discussion_r1023512856